app/src/main/java/com/dessalines/thumbkey/keyboards/DETypeSplit.kt

```diff
@@ -559,7 +559,7 @@ val KB_DE_TYPESPLIT_SHIFTED = KeyboardC(
 )
 
 val KB_DE_TYPESPLIT: KeyboardDefinition = KeyboardDefinition(
-    title = "DE Type-Split deutsch",
+    title = "deutsch type-split",
```

Comment:
tbh personally i liked the user of capital letters, it makes it easier to distinguish parts of the title than everything in small letters.

Reply:
I did a few, but they did not look good, mainly because the official names of most languages do not use upper-case.